modules/overview-of-llama-stack.adoc

```diff
@@ -25,10 +31,11 @@ endif::[]
 
 Llama Stack includes the following components:
 
+* **Integration with {productname-short}** by using the `LlamaStackDistribution` custom resource, simplifying configuration and deployment.
 * **Inference model servers** such as vLLM, designed to efficiently serve large language models.
 * **Vector storage** solutions, primarily Milvus, to store embeddings generated from your domain data.
```

Comment:
Same comment here: LLS does *not* include a vector database, is can connect and use an _existing_ vector database.